core/client/fs/src/main/java/alluxio/client/file/AlluxioFileInStream.java

```diff
@@ -233,6 +234,11 @@ private int positionedReadInternal(long pos, byte[] b, int off, int len) throws
       return -1;
     }
 
+    LOG.info("AMDEBUG pread len {} ", len);
+    if (len < 2 * Constants.MB) {
+      LOG.info("AMDEBUG set for pread len {} ", len);
+      mOptions.setPositionShort(true);
```

Comment:
Does this option only apply to positioned reads or also reads through the `read` API?

Reply:
only applies to `pread`. reads through the `read` API always are assumed to be sequential by this heuristic